com.github.marhkb.Pods.Devel.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "com.github.marhkb.Pods.Devel",
```

Comment:
it should be the "release" app-id.

Reply:
Done.
One thing that is unclear to me:
I had assumed that I need to change it later back to `*.Devel` on the beta branch to make beta releases distinguishable from production builds. But other manifests I found in beta branches don't do this ([Example](https://github.com/flathub/org.gnome.Fractal/tree/beta)). Does the build bot take care of this?

Reply:
definitely not. It won't work. The app-id is the app-id.

Also .Devel is not for beta, Devel is for development. Flathub isn't for development.

Reply:
So I guess it's not possible for users to install a stable and a beta version at the same time, because the app-id is not different?

Reply:
yes it is. Different branches. They'll share the config directory too.